geode-gfsh/src/main/java/org/apache/geode/management/cli/GfshCommand.java

```diff
@@ -59,6 +59,14 @@ public boolean isOnlineCommandAvailable() {
     return gfsh.isConnectedAndReady();
   }
 
+  /**
+   * For those commands that could possibly change the cluster configuration, they need to
+   * override this method to return true.
+   */
+  public boolean affectsClusterConfiguration() {
```

Comment:
How about naming it as "updatesClusterConfiguration" (); they both are same, just another name to consider, its left to you.